### PR TITLE
WIP: README updates and an environment.yml

### PR DESCRIPTION
**Pull request by hunterowens**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,6 +95,15 @@ generating backfill runs based on the DAG's configured start date, which could g
 (set `schedule_interval=None` in your DAG definition to prevent these scheduled runs).
 You'll likely want to toggle the DAG back to "Off" as soon as your desired task starts running.
 
+#### Pre Commit Install
+
+Please note, this project uses Black as a code formatter. To setup the pre-commit installs, run
```
Comment (ian-r-rose) [inline README.md]: The pre-commit hooks also runs flake8 and isort, FWIW

File: environment.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+name: aqueduct
+channels:
+  - conda-forge
+dependencies:
+    - python=3.6
```
Comment (ian-r-rose) [inline environment.yml]: I think these should have extremely strict versions (as per #124).

**Comment by hunterowens** [comment]
I think we should close this PR, at list for now. 

Once the aqueduct repo is truly dead, I think per ETL specific dockerfiles will be the way to go.

**Comment by hunterowens** [comment]
Not repo, but rather, `dags folder`

**Pull request closed without merging**